content/tutorials/portenta-h7/por-ard-lvgl/content.md

```diff
@@ -20,7 +20,7 @@ Graphical User interfaces are necessary for visualising information and interact
 
 # Building a simple GUI 
 
-we are going to work on the **Portenta_lvgl** example , The Portenta H7 comes with an inbuilt,  In this tutorial we are going to upload the lvgl example to the portenta board. Once the sketch is uploaded the board becomes a usb host where it then is connected to the usb hub which then is connected to an external monitor and powered externally. when connected the monitor the buttons and text fields will be connected. 
+We are going to build a sketch from scratch,  In this tutorial we are going to upload our sketch to the portenta board. Once the sketch is uploaded the board becomes a usb host where it then is connected to the usb hub which then is connected to an external monitor and powered externally. when connected the monitor the buttons and text fields will be connected. 
```

Comment:
This tutorial will guide you to build a basic user interface using the LVGL and the USBhost Libraries that you will have download using the Arduino Library Manager. The setup for this tutorial requires you to first to upload the finished sketch file to the Portenta board where it converts the board into a usb host Device. You will then connect the board to a USB-hub and connect an external monitor. Once the HUB is powered externally, a graphical user interface with a button and a text-field will be displayed on the screen.

Reply:
you can replace the text with what I wrote above